docs/docs/user_management/user_roles.md

```diff
@@ -1,3 +1,10 @@
 # User Roles in Minder
 
-Explains user roles
+When incorporating a user into your project, it's crucial to assign them the appropriate role based on their responsibilities and required access levels.
+Minder currently offers the following roles:
+
+- `viewer`: Provides read-only access to the project. Users with this role can view associated resources such as enrolled repositories, rule types, and profiles.
+- `editor`: Grants the same permissions as the viewer role, along with the ability to edit project resources, excluding the project itself and the list of providers.
```

Comment:
How about "in addition to the viewer permissions, editors can author profiles and rule types, as well as add resources to manage.  Editors _cannot_ enroll additional providers or change or delete projects."